sharing_configs/admin.py

```diff
@@ -38,55 +39,60 @@ class SharingConfigsExportMixin:
     change_form_export_template = "sharing_configs/admin/export.html"
     sharing_configs_export_form = ExportToForm
 
-    def get_sharing_configs_export_data(self, obj: object) -> Union[str, bytes]:
+    def get_sharing_configs_export_data(self, obj: object) -> bytes:
         """
-        Derived class should provide object to export
+        Derived class should override this method for export converting model object into bytes
 
         """
         raise NotImplemented
 
     def sharing_configs_export_view(self, request, object_id):
         """
-        return template with form for GET request;
-        process form data from POST request and make API call to endpoint
+        process form data from POST request and make API call to endpoint;
+        initial expects str representation of an object
         """
         info = (
             self.model._meta.app_label,
             self.model._meta.model_name,
         )
+        main_url = f"admin:{info[0]}_{info[1]}_export"
+        ajax_url = f"admin:{info[0]}_{info[1]}_ajax"
```

Comment:
Why do you need `ajax_url` here?

Reply:
Yes, ajax is servered by another method == get_ajax_fetch_files

Reply:
`get_ajax_fetch_files` is not included in this class, please remove this variable here to avoid confusion

Reply:
I'm not sure. This url should be present at the moment when the form gets rendered. How can I include this url otherwise in the template? Thx.

Reply:
I don't see it in the template. Could you give me a link to the code line where this url is used in the template?

Reply:
sharing_configs.templates.import.html
`<form id="import-form" action="{% url context.main_url %}" data-action="{% url context.ajax_url %}" method="POST"`

Reply:
Oh, I set it also in export. Will fix it